index.html

```diff
@@ -0,0 +1,324 @@
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <div class="form-container">
+        <form action="https://formspree.io/f/mwkaywog" method="post">
+          <h1 class="form-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+          <div class="form-group">
+            <div class="form-control">
+              <label>
+                  <input type="text" name="full_name" placeholder="Full name" required>
+              </label>
+            </div>
+
+            <div class="form-control">
+              <label>
+                  <input type="email" name="email" placeholder="Email address" required>
+              </label>
+            </div>
+            <div class="form-control">
+              <textarea name="message" cols="30" maxlength="500" rows="10" required></textarea>
+            </div>
+            <button type="submit" class="form-button">Get in touch</button>
+          </div>
+        </form>
+      </div>
+    </footer>
```

Comment:
- [ ] Optional: Kindly also add the social links section from the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

![Screenshot from 2022-03-10 01-23-06](https://user-images.githubusercontent.com/52934128/157562756-f637638a-8d9e-4ae2-80c1-79426378b702.png)